image/config.go

```diff
@@ -68,6 +64,44 @@ func findConfig(w walker, d *descriptor) (*config, error) {
 	}
 }
 
+func (c *config)validate() error {
```

Comment:
Please Go format your code.

@vbatts @philips Do we have a Go style guide in place?